### ISSUE TITLE
Replace additional characters

### ISSUE THREAD

**Issue by randoogle** [chop-dbhi/vscode-multiline-string-editor#11]
I often have to maintain json that gets exported from another third-party application. In addition to new lines "\n" it also include carriage returns "\r" (in a windows style line ending I guess).
Some of the code also includes tabs "\t" and of course double-quotes \\"

I've gone ahead and made the changes on my end and re-built the extension, and it's working great now.
I know there's probably other characters that might need to be included, but I feel like I've covered the most of the use cases.

**Comment by randoogle** [comment]
Hmm, github isn't letting me upload my patch file, so I'll upload it as a txt.
[additional_characters.txt](https://github.com/chop-dbhi/vscode-multiline-string-editor/files/11216894/additional_characters.txt)

**Comment by murphyke** [comment]
@randoogle Thanks for the contribution. I've been meaning to update the dependencies for this extension, so I can integrate your changes at the same time. Honestly it will probably be a few weeks ....

**Comment by randoogle** [comment]
Yeah, I noticed there were some outdated dependency issues during the install/compile process.
Take your time. I'm just glad this is even being maintained -- so thanks for that. 👍
Let me know if you need anything else. Honestly, this is my first time trying to contribute actual code to any project, and I wasn't even sure the right way to generate/send a patch.

**Comment by jefflord** [comment]
@murphyke  "Honestly it will probably be a few weeks..." so, how's the patch coming?  Any plans to use the @randoogle patch?
